apps/frontpage/components/docs/footer/actions.ts

```diff
@@ -0,0 +1,625 @@
+'use server';
+
+import { z } from 'zod';
+import dedent from 'dedent';
+import fetch from 'node-fetch';
+
+import { headers } from 'next/headers';
+import type { TreeProps } from '@repo/utils';
+import { docsVersions } from '@repo/utils';
+import { generateDocsTree } from '../../../lib/get-tree';
+import { language } from 'gray-matter';
+
+export async function sendFeedback(
+  prevState: {
+    message: string;
+  },
+  formData: FormData,
+) {
+  const schema = z.object({
+    feedback: z.string().min(1),
+    reaction: z.string(),
+    slug: z.string(),
+    renderer: z.string(),
+    language: z.string(),
+    version: z.string(),
+  });
+
+  const parse = schema.safeParse({
+    slug: formData.get('slug'),
+    feedback: formData.get('feedback'),
+    reaction: formData.get('reaction'),
+    renderer: formData.get('renderer'),
+    language: formData.get('language'),
+    version: formData.get('version'),
+  });
+
+  if (!parse.success) {
+    return { message: 'Failed' };
+  }
+
+  const data = parse.data;
+
+  const pat = process.env.GITHUB_STORYBOOK_BOT_PAT;
+  if (!pat) {
+    throw new Error('GITHUB_STORYBOOK_BOT_PAT not found in environment');
+  }
+
+  const versions = docsVersions.map(({ id }) => id);
+
+  const slugs = new Set<string>([]);
+
+  function addSlugs(tree: TreeProps[]) {
+    tree.forEach((node) => {
+      if (node.type === 'directory') {
+        addSlugs(node.children!);
+      } else {
+        slugs.add(node.slug);
+      }
+    });
+  }
+
+  versions.forEach((v) => {
+    const tree = generateDocsTree(`content/docs/${v}`);
```

Comment:
This doesn't work, because that location doesn't exist:

```
Error: ENOENT: no such file or directory, scandir 'content/docs/8.0-test-1'
```
— [Vercel log](https://vercel.com/storybookjs/web/logs?slug=app-future&slug=en-US&slug=storybookjs&slug=web&slug=logs&page=1&timeline=past30Minutes&startDate=1717963701744&endDate=1717965501744&selectedLogId=1717965490049549004920600000&selectedLogTimestamp=1717965490049&forceEndDate=1717965490049)

It may only be a matter of using `path` & `__dirname`?